assets/built-in/transformers/kubernetes/argocd/transformer.yaml

```diff
@@ -17,3 +17,6 @@ spec:
   produces:
     KubernetesYamls:
       disabled: false
+  config:
+    outputPath: "deploy/cicd/argocd"
+    useDefaultValuesInYamls: false
```

Comment:
would `setDefaultValuesInYamls` be a better name?

Reply:
Yes sure

Reply:
Done.